packages/vscode-extension/src/commandsTreeViewProvider.ts

```diff
@@ -143,9 +144,24 @@ class TreeViewManager {
         undefined,
         undefined,
         { name: "publish", custom: true }
-      )
+      ),
     ];
 
+    if (
+      true /*await exp.getExpService().getTreatmentVariableAsync("flight", "treatmentVariable", true)*/
```

Comment:
always true?

Reply:
Because the flight name and treatment variable hasn't been confirmed, so will update later